dotnet-test-samples/async-architectures/async-lambda-dynamodb/tests/AsyncTesting.IntegrationTest/AsyncTests.cs

```diff
@@ -0,0 +1,120 @@
+namespace AsyncTesting.IntegrationTest;
+
+using System.Text;
+
+using Amazon.DynamoDBv2.Model;
+using Amazon.S3.Model;
+
+using FluentAssertions;
+
+using Xunit.Abstractions;
+
+public class AsyncTests : IClassFixture<Setup>
```

Comment:
I would usually use a less generic name for my test, I think that we have an e2e test here and not integration test